matrix_sdk_base/src/client.rs

```diff
@@ -129,7 +129,7 @@ fn hoist_member_event(
     Ok(e)
 }
 
-fn hoist_room_event_prev_content(
+pub fn hoist_room_event_prev_content(
```

Comment:
This will now need docs since it's public, you can mostly copy them from the other methods.